test/ConfigDiscoveryTest.php

```diff
@@ -87,24 +95,24 @@ public function createModulesConfig()
     public function assertOptionsContainsNoopInjector(Collection $options)
     {
         if ($options->isEmpty()) {
-            throw new ExpectationFailedException('Options array is empty; no NoopInjector found!');
+            $this->fail('Options array is empty; no NoopInjector found!');
```

Comment:
In my merge branch, I added a file `test/_autoload.php` that does a `class_alias` of the non-namespaced version to the namespaced variant, and added an `autoload-dev.files` entry for this file. Works like a charm!